.travis.yml

```
@@ -3,9 +3,9 @@ node_js: "node"
 os:
   - linux
 python:
+  - "3.7"
```

I think 3.7 is still not standard on Travis.

```suggestion
  - "3.7-dev"
```

---

The build works on 3.7, should I then just leave it as is? Or is the `3.7-dev` special in a way beyond that?

---

`-dev` installs the latest release candidate so it is a bit different. However, if 3.7 now exists Please ignore this suggestion :smiley: